HCK16_2023_Granada_RH/README.md

```diff
@@ -1,77 +1,106 @@
-# NeuroDataReHack 2
+# NeuroDataReHack 2023
 ## *Generate new insights from existing neurophysiology data through secondary analysis*
 
 * [Dates and Location](#dates-and-location)
+* [Objective](#objective)
 * [Application](#application)
-<!-- * [Logistics](#logistics) -->
+* [Logistics](#logistics)
 * [Organizing Committee](#organizing-committee)
-    * [Additional Organizational Support](#additional-organizational-support)
 * [Resources](#resources)
 * [What to bring?](#what-to-bring)
-* [Objective](#objective)
 * [Schedule](#schedule)
-* [Projects](#projects)
-    * [Creating a New Project](projects/README.md)
 * [Disclaimer](#disclaimer)
 
 ## Dates and Location
 
 - **Dates:** September 5-8, 2023
-- **Location:** Granada, Spain (exact location TBD)
+- **Location:** [Hotel Andalucia Center in Granada, Spain](https://www.hotelescenter.es/en/hotel-andalucia-center-granada/)
+  ([google maps](https://goo.gl/maps/rufpp3g39Aefnj289))
 
-## Application
+## Objective
+[The DANDI Archive](http://dandiarchive.org) now has 110+ neurophysiology datasets in the 
+[Neurodata Without Borders](http://nwb.org) format  spanning many species, brain  areas, task types, and imaging 
+modalities. These include high-value datasets, e.g. from The Allen Institute, the MICrONS project, and the 
+International Brain Laboratory, as well as diverse contributions from neuroscience labs around the world. In this 
+workshop, we will teach attendees about the open neurophysiology datasets available on the DANDI Archive and train 
+them on how to maximally utilize the archive and the NWB standard to incorporate existing data into their scientific 
+workflows. Feedback from attendees will be used to improve the software and data standard to better enable 
+reanalysis workflows.
 
-Apply to attend Neurodata Rehack 2 [here]
-<!--
-## Logistics
+Following the event, participants will be invited to apply for a Kavli Foundation Neurodata Discovery Award, which 
+awards $50,000 of funding to continue data reanalysis projects that come out of the NeuroDataReHack event.
 
-**Housing:**
+This event is held as a satellite of the [IBRO World Conference 2023](http://ibro2023.org) with the goal of making 
+it more accessible to diverse participants who might not otherwise have the opportunity to participate in this
+workshop. Attendance of IBRO 2023 is not a requirement for application.
 
-**Travel:**
+This event will primarily focus on analyzing existing data in NWB and on DANDI, **not** converting data to NWB. If 
+you are interested in learning how to convert data, consider signing up for an NWB User Days event.
 
--->
+## Application
 
-## Organizing Committee
+Space for the event is limited. Apply to attend NeuroDataReHack 2023 [here](https://forms.gle/akaThWFBJrv7udev9). 
```

Comment:
In the application, regarding the question "Please propose a potential project to derive scientific insight by analyzing one or more specific dandisets on the DANDI Archive (dandiarchive.org)." I recommend providing some guidelines about how long of a response we expect. A couple sentences? Max 200 words?

Reply:
Good point. Let's say 250 words